### DISCUSSION TITLE
useHistory

### DISCUSSION THREAD

**Discussion by muskanthapa2000** [community/community#66456] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

useHistory is also not working

**Comment by divyam751** [comment]
useHistory is deprecated in newer version of React-router-dom, you can use  "useNavigate" instead of useHistory.

**Comment by Akash1134** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
